lib/common/api/net-client-request.ts

```diff
@@ -427,9 +427,8 @@ export class ClientRequest extends Writable implements Electron.ClientRequest {
     this._started = true;
     const stringifyValues = (obj: Record<string, { name: string, value: string | string[] }>) => {
       const ret: Record<string, string> = {};
-      for (const k of Object.keys(obj)) {
-        const kv = obj[k];
-        ret[kv.name] = kv.value.toString();
+      for (const { name, value } of Object.values(obj)) {
```

Comment:
```suggestion
      for (const [ name, value ] of Object.entries(obj)) {
```

I think this was what was intended? `Object.values` returns an array of just the values.

It looks like this isn't covered by any tests as CI is passing. 😅 We should look into ensuring this code path is covered by tests.

Reply:
Ah, you're right that I misread it. Unexpected way of doing things. 🤔